pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -215,9 +215,15 @@ public static class ControllerPeriodicTasksConf {
         "controller.realtime.segment.deepStoreUploadRetryEnabled";
     public static final String DEEP_STORE_RETRY_UPLOAD_TIMEOUT_MS =
         "controller.realtime.segment.deepStoreUploadRetry.timeoutMs";
+    public static final String ENABLE_TMP_SEGMENT_ASYNC_DELETION =
+        "controller.realtime.split.commit.segment.tmp.cleanup.async.enable";
```

Comment:
The tmp file is not generated only for split commit case. Suggest renaming the keep to match other configs
```suggestion
        "controller.realtime.segment.tmpFileAsyncDeletionEnabled";
```